samples/python/tutorial_code/imgProc/BasicGeometricDrawing/basic_geometric_drawing.py

```diff
@@ -6,10 +6,10 @@
 def my_ellipse(img, angle):
     thickness = 2
     line_type = 8
-
+    
```

Comment:
Please try to remove these trailing whitespace.

Reply:
White spaces removed.